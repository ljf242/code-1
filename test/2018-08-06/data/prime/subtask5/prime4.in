60145790186139 60145790286134 1000000000
