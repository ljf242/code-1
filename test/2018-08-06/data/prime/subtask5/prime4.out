60145790260495
